### PR TITLE
Document

### PR DESCRIPTION
**Pull request by TehanovAnton**
#9 I create Document and CarOwnerDocument models, add associations. User can have many documents

### CODE REVIEW

**Comment by TehanovAnton** [comment]
#9 Add business models.
I create Document and CarOwnerDocument models, add association. User can have many documents

**Comment by dmokreckiy** [comment]
@TehanovAnton same as previous PR - description, reviewers, assignee.

File: db/migrate/20211231104737_create_documents.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateDocuments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :documents do |t|
+      t.string :name, null: false
+
```
Comment (dmokreckiy) [inline db/migrate/20211231104737_create_documents.rb:6]: `term_of_validity`? `срок действия документа` is always the same on each type of the document

File: db/migrate/20211231105140_create_car_owner_documents.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CreateCarOwnerDocuments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :car_owner_documents, id: false do |t|
```
Comment (dmokreckiy) [inline db/migrate/20211231105140_create_car_owner_documents.rb]: Join table? Naming?

**Review by dmokreckiy — CHANGES_REQUESTED** [review]
Please check my comments

**Review by yury-yaroshyk-itechart — APPROVED** [review]

**Pull request merged**